src/components/CheckinNoteModal/CheckInNoteModal.test.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import {
+  render,
+  cleanup,
+  fireEvent,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+
+import {
+  Modal,
+  MultiColumnList,
+} from '@folio/stripes/components';
+
+import CheckinNoteModal from './CheckinNoteModal';
+
+const cancelLabel = 'Cancel Label';
+const confirmLabel = 'Confirm Label';
+const message = 'test message';
+const labelIds = {
+  cancel: 'ui-checkin.statusModal.cancel',
+  confirm: 'ui-checkin.statusModal.confirm',
+};
+
+const initialProps = {
+  open: true,
+  hideCancel: false,
+  hideConfirm: false,
+  id: 'id',
+  notes: [],
+  columnMapping: {},
+  columnWidths: {},
+  heading: 'Test Heading',
+  onConfirm: jest.fn(),
+  onCancel: jest.fn(),
+  visibleColumns: ['test'],
+  cancelLabel,
+  confirmLabel,
+  message,
+};
+
+const renderModal = (props = initialProps) => {
+  return render(
+    <CheckinNoteModal
+      {...props}
+    />
+  );
+};
+
+describe('CheckinNoteModal', () => {
+  afterEach(cleanup);
+
+  describe('Initial render with default properties', () => {
+    let container;
+
+    beforeEach(() => {
+      container = renderModal();
+    });
+
+    it('CheckInNoteModal should get correct props', () => {
+      const expectedProps = {
+        size: 'small',
+        open: initialProps.open,
+        label: initialProps.heading,
+        id: initialProps.id,
+        dismissible: true,
+        'data-test-check-in-note-modal': true,
```

Comment:
We have agreement don't check styles and component test ids (data-test-check-in-note-modal - this is test id for bigtest).
We should left here `id` because we pass it directly.
Also add please `onClose` in expectedProps.